cli/index.ts

```diff
@@ -614,7 +645,27 @@ export function runCli() {
               }
             })
           );
+
           printFormatFilesResult(results);
+
+          // Return error code if there are any formatting errors
+          const failedFormatResults = results.filter(result => !!result.err);
+          if (failedFormatResults.length > 0) {
+            return 1;
```

Comment:
Could we please add the following for consistency:

```
printError(`${failedFormatResults.length} file(s) failed to format.`);
```
The files themselves should be already printed as we discussed in the thread.